OpenRiaServices.DomainServices.Client/Test/Silverlight/Data/EntitySetTests.cs

```diff
@@ -6,7 +6,7 @@
 using System.ComponentModel;
 using System.Globalization;
 using System.Linq;
-using DataTests.AdventureWorks.LTS;
+using DataTests.Northwind.LTS;
```

Comment:
Why did this namespace change?

Reply:
The AdventureWorks namespace was not being used, and caused conflicts when I added the InferredAddThenAttach test (since the AdventureWorks namespace also has a Product class).